components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/bean/APIsForThrottleStatsValue.java

```diff
@@ -23,7 +23,7 @@
 public class APIsForThrottleStatsValue {
     private int success_request_count;
     private int throttle_out_count;
-    private List<String> api_apiPublisher_applicationName_facet;
+    private List<String> api_year_month_week_day_facet;
```

Comment:
yes. this class is used to parse the JSON response back to java class which contain 'api_year_month_week_day_facet" as a attribute. In that case variable should follow same name. do we have options?